ext/scheduler/airflow/__lib.py

```diff
@@ -311,6 +311,15 @@ def _xcom_value_has_error(_xcom) -> bool:
         log.info(f'failures: {failure_message}')
     
     task_instance = context.get('task_instance')
+    
+    if event_meta["event_type"] == "TYPE_FAILURE" :
+        dag_run = context['dag_run']
+        tis = dag_run.get_task_instances()
+        for ti in tis:
+            if ti.state == "failed":
```

Comment:
nit: use `State.FAILED` instead of plain string https://github.com/apache/airflow/blob/614858fb7d443880451e6111b27fdaf942f563a4/airflow/utils/state.py#L82

other than that, LGTM

Reply:
sure, fixed